RiftWizardOverrides.py

```diff
@@ -139,13 +141,17 @@ def get_repeatable_keys(self):
 		RiftWizard.KEY_BIND_UP_RIGHT,
 		RiftWizard.KEY_BIND_UP_LEFT,
 		RiftWizard.KEY_BIND_DOWN_RIGHT,
-		RiftWizard.KEY_BIND_DOWN_LEFT
+		RiftWizard.KEY_BIND_DOWN_LEFT,
+		RiftWizard.KEY_BIND_PASS

```

Comment:
How come PASS is repeatable? Unless it’s changed, only movement keys are repeatable in vanilla. Having pass be repeatable sounds dangerous

Reply:
Check the most recent source code, PASS is repeatable.